### PR TITLE
[Components with more than 1 service] LPS-192496 Refactor ConfigurationModelIndexer component

### PR DESCRIPTION
**Pull request by Valmir-unicap**
Related: https://liferay.atlassian.net/browse/LPS-192496

### CODE REVIEW

**Comment by Valmir-unicap** [comment]
ci:test:startup

**Comment by Valmir-unicap** [comment]
ci:test:usm

File: modules/apps/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/search/ClusterConfigurationModelIndexer.java [hunk 1]
```diff
@@ -53,7 +68,7 @@ public void initialize() throws PortletException {
 			}
 
 			if (_clusterMasterExecutor.isMaster()) {
-				_bundleTracker = _configurationModelIndexer.initialize();
+				_bundleTracker = _initialize();
```
Comment (rafaprax) [inline modules/apps/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/search/ClusterConfigurationModelIndexer.java:71]: Hey @Valmir-unicap, please initialize the `_bundleTracker` into the `_initialize` method and make the method `void`. Thx

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by Valmir-unicap** [comment]
ci:test:sf

**Comment by Valmir-unicap** [comment]
ci:test:relevant

**Comment by Valmir-unicap** [comment]
ci:test:startup

**Comment by Valmir-unicap** [comment]
ci:test:usm

**Comment by Valmir-unicap** [comment]
ci:stop

**Comment by Valmir-unicap** [comment]
ci:test:startup

**Comment by Valmir-unicap** [comment]
Yesterday the `ci` was broken. 
Thread: https://liferay.slack.com/archives/C19PWQ74J/p1691370995470499

**Review by rafaprax — APPROVED** [review]

**Comment by catarineaguiar** [comment]
no regressions found

**Comment by tinatian** [comment]
@Valmir-unicap @rafaprax 
Please do :
1. Move clusterConfigurationModelIndexer.initialize(); from SearchResultsMVCRenderCommand to ConfigurationModelIndexer#search
2. Merge ClusterConfigurationModelIndexer and ConfigurationModelIndexer
3. Register IdentifiableOSGiService manually.

Please resend.
Thanks.


Tina.

**Pull request closed without merging**